blockchain/ethereum/ethereumtest/chain.go

```diff
@@ -56,9 +56,10 @@ func NewChainBackendSetup(t *testing.T, rng *rand.Rand, numAccs uint) *ChainBack
 	cbEth := newSimContractBackend(t, walletSetup.Accs, walletSetup.Keystore)
 	cb := &internal.ChainBackend{Cb: &cbEth, TxTimeout: ChainTxTimeout}
 
-	adjudicator, err := cb.DeployAdjudicator()
+	addr := walletSetup.Accs[0].Address()
+	adjudicator, err := cb.DeployAdjudicator(addr)
 	require.NoError(t, err)
-	asset, err := cb.DeployAsset(adjudicator)
+	asset, err := cb.DeployAsset(adjudicator, addr)
```

Comment:
```suggestion
	asset, err := cb.DeployAsset(adjudicator, onChainAddr)
```

Reply:
[Renamed](https://github.com/hyperledger-labs/perun-node/pull/114/commits/aebcf94494c5d13ad272ecadd28edfe7b22dc659) the variable.